src/plugins/intel_cpu/src/emitters/snippets/input_repacker.hpp

```diff
@@ -28,7 +28,7 @@ struct InputRepacker {
                   VectorDims in_offsets,
                   VectorDims out_offsets);
 
-    template <class T = InputRepackerKernel, std::enable_if_t<std::is_base_of_v<InputRepackerKernel, T>, bool> = true>
+    template <class T = InputRepackerKernel, typename = std::enable_if_t<std::is_base_of_v<InputRepackerKernel, T>>>
```

Comment:
Optional alternative:
```suggestion
    template <class T = InputRepackerKernel, std::enable_if_t<std::is_base_of_v<InputRepackerKernel, T>>* = nullptr>
```